modules/telegramBot/module.go

```diff
@@ -0,0 +1,15 @@
+package telegramBot
+
+import (
+	"go.uber.org/fx"
+	"os"
+)
+
+var Module = fx.Options(
```

Comment:
```suggestion
var Module = fx.Module("telegram_bot",
```
В новых версиях `uber-go/fx` появились модули, лучше использовать их
Мы в нашем проекте тоже начали их применять
https://uber-go.github.io/fx/modules.html#writing-modules